package.json

```diff
@@ -62,11 +62,11 @@
     "traverse": "^0.6.6"
   },
   "peerDependencies": {
-    "@grpc/grpc-js": "^1.2.0",
```

Comment:
can we leave this to `^1.2.0` please. There is no reason to strictly require a specific patch version of core grpc package.

Reply:
No problem  👍